src/grisp_connect_api.erl

```diff
@@ -39,12 +56,36 @@ handle_rpc_messages([{internal_error, _, _} = E | Batch], Replies) ->
     handle_rpc_messages(Batch,
                         [grisp_connect_jsonrpc:format_error(E)| Replies]).
 
-handle_request(<<"post">>, #{type := <<"flash">>} = Params, ID) ->
+handle_request(?method_post, #{type := <<"start_update">>} = Params, ID) ->
+    try 
+        URL = maps:get(url, Params),
+        Reply = case start_update(URL) of
+            {error, grisp_updater_unavailable} ->
+                {error, -10, grisp_updater_unavailable, undefined, ID};
+            {error, already_updating} ->
+                {error, -11, already_updating, undefined, ID};
+            {error, Reason} ->
+                ReasonBinary = iolist_to_binary(io_lib:format("~p", [Reason])),
+                grisp_connect_jsonrpc:format_error({internal_error, ReasonBinary, ID});
+            ok ->
+                {result, ok, ID}
+        end,
+        {request, grisp_connect_jsonrpc:encode(Reply)}
+    catch
+        throw:bad_key ->
+            {request, 
+             grisp_connect_jsonrpc:format_error(
+                {internal_error, invalid_params, ID})}
+        end;
+handle_request(?method_post, #{type := <<"flash">>} = Params, ID) ->
```

Comment:
Maybe we want this to be overridable later on with local configuration. The app developers may want to control the led and don't want us to mess up with them. But this is for later...